piet-wgsl/examples/winit/src/main.rs

```diff
@@ -103,5 +104,28 @@ async fn run() -> Result<()> {
 }
 
 fn main() {
-    pollster::block_on(run()).unwrap();
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        pollster::block_on(run());
+    }
+    #[cfg(target_arch = "wasm32")]
+    {
+        let event_loop = EventLoop::new();
+        let window = winit::window::Window::new(&event_loop).unwrap();
+
+        std::panic::set_hook(Box::new(console_error_panic_hook::hook));
+        console_log::init().expect("could not initialize logger");
+        use winit::platform::web::WindowExtWebSys;
+
+        // On wasm, append the canvas to the document body
+        let canvas = window.canvas();
+        canvas.set_width(1044);
```

Comment:
I see this size is pre-existing, so I don't think we need to change it, but `1044` is not a very round number

Reply:
This was specifically not very round to test an output that didn't align to a tile boundary.